_data/wgs.yml

```diff
@@ -2,68 +2,70 @@
   bottomliner: Farbod Saraf
   slug: leadership
   status: Incubating
-  ranking: 1
+  ranking: 0
```

Comment:
This hasn't had it's first meeting yet. Keep it in incubating.

Reply:
@RichardLitt ah, my bad, we should document that.

Reply:
Hmm. Document what? That it hasn't had a meeting yet? It currently says that on the website. Do you want me to document it, somewhere else?

Reply:
nah all good. My bad!